app/models/loader/types.rb

```diff
@@ -199,8 +199,18 @@ def create!
     class User < Record
       def_delegators :@policy_object, :public_keys, :account, :role_kind, :uidnumber, :restricted_to
 
+      def check_user_creation_allowed(res_id:)
+        user_creation_allowed = ENV.fetch('CONJUR_ALLOW_USER_CREATION', 'true').downcase
+        if user_creation_allowed == 'false' && res_id.include?('@')  # not under root
+          message = "User creation is disallowed - please address administator"
```

Comment:
This reads a bit rough.  What about something like:
```
User creation through policy is disabled. 
```

Reply:
done